src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1310,5 +1310,52 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         {
             Interop.Window.delete_Window(swigCPtr);
         }
+
+        /// <summary>
+        /// FrameCallbackType
+        /// </summary>
+        /// <param name="frameId"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void FrameCallbackType(int frameId);
+
+        /// <summary>
+        /// AddFrameRenderedCallback
+        /// </summary>
+        /// <param name="callback"></param>
+        /// <param name="frameId"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void AddFrameRenderedCallback(FrameCallbackType callback, int frameId)
+        {
+            IntPtr ip = Marshal.GetFunctionPointerForDelegate<Delegate>(callback);
+
+            if (IntPtr.Zero == ip)
+            {
+                throw new ArgumentNullException(nameof(callback), "FrameCallbackType should not be null");
+            }
+
+            Interop.WindowInternal.AddFrameRenderedCallback(swigCPtr, new HandleRef(this, ip), frameId);
+
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// AddFramePresentedCallback
+        /// </summary>
+        /// <param name="callback"></param>
+        /// <param name="frameId"></param>
```

Comment:
/**
   * @brief Adds a callback that is called when the frame is displayed on the display.
   *
   * @param[in] callback The function to call
   * @param[in] frameId The Id to specify the frame. It will be passed when the callback is called.
   *
   * @note A callback of the following type may be used:
   * @code
   *   void MyFunction( int frameId );
   * @endcode
   * This callback will be deleted once it is called.
   *
   * @note Ownership of the callback is passed onto this class.
   */